### ISSUE TITLE
setting `registry.consul.tagprefix` doesn't seem to work

### ISSUE THREAD

**Issue by far-blue** [fabiolb/fabio#825]
I'm trying to use the `registry.consul.tagprefix` config value to change the default `urlprefix-` to `fabio-`. I have services in consul passing health checks and registering service tags of the form `fabio-...` but when I run a Fabio instance with the tag prefix set to `fabio-` it doesn't find any services.

I think the issue might be here: 
https://github.com/fabiolb/fabio/blob/231e7f24a13fde2fca2ffd9e8f553c18addac165/demo/server/server.go#L207

It looks like it is hard-coded to use `urlprefix-` when calculating http server services from the service tags.

**Comment by ketzacoatl** [comment]
@far-blue does it work if you use the default? I'm successfully using this feature in all my deployments, even with the latest stable release (v1.6.0). Are there any logs in fabio related to the tags or the config?

**Comment by far-blue** [comment]
no, no logs. Everything appears to be working correctly but fabio simply doesn't "see" any services to route.

**Comment by nathanejohnson** [comment]
@far-blue we're using this feature in production as well (overriding registry.consul.tagprefix for some fabios).  Can you help come up with a minimal environment that demonstrates this issue?  can you try setting the log level to TRACE ?  That might shed some light.
